README.md

```diff
@@ -82,9 +82,21 @@ val configuration = FaceCaptureConfiguration(
 ```
 
 #### Face center
-The face center is a `PointF` representing the expected position of the center of the captured face.
+The face center is a `PointF` representing the expected position of the centre of the user's face inside the `FaceCapture` view. In the samples below, it is represented by the intersection of the red and blue lines in the sample images.
```

Comment:
Aren't these centre positions relative to the full image rather than the capture View? Relevant because parts of the image may be cropped if the aspect ratios are different.

Also I guess the American spelling of `center` is more consistent with our config option and the rest of this documentation